pycon/static/less/site.less

```diff
@@ -391,6 +391,7 @@ aside {
         -moz-transform: translate(-50%,0);
         -ms-transform: translate(-50%,0);
```

Comment:
It appears that when I remove the `transform` attributes and both `margin-left` and `margin-right` it works perfectly, tested on Chrome and Firefox on Linux. Will update.